### PR TITLE
Add backendContext in Web env.

### PR DESCRIPTION
**Pull request by zenoslin**
- [Add functions to make PAGImage from texture or pixels.](https://github.com/Tencent/libpag/commit/28a278dc218ddd16a0e529c310e14ea746bebb2a)
- [Packaging framebuffer in Web env.](https://github.com/Tencent/libpag/commit/d3b197484de7b8f8822ab8fb414b8812b9f5f7fe)

### CODE REVIEW

File: web/src/core/backend-context.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  /**
+   * Get WebGLTexture by handle.
+   */
+  public getTexture(handled: number): WebGLTexture | null {
+    return BackendContext.module.GL.textures[handled];
+  }
+  /**
+   * Delete WebGLTexture reference in EmscriptenGL.
+   */
+  public deleteTexture(handle: number) {
```
Comment (domchen) [inline web/src/core/backend-context.ts:79]: 这个用unregisterTexture是不是更好？delete的话会让外部以为不用再调用glDeleteTextures?

File: web/src/core/backend-context.ts [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  /**
+   * Get WebGLFramebuffer by handle.
+   */
+  public getRenderTarget(handle: number): WebGLFramebuffer | null {
+    return BackendContext.module.GL.framebuffers[handle];
+  }
+  /**
+   * Delete WebGLTexture reference in EmscriptenGL.
+   */
+  public deleteRenderTarget(handle: number) {
```
Comment (domchen) [inline web/src/core/backend-context.ts:97]: 同上

File: web/src/core/backend-context.ts [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    BackendContext.module.GL.framebuffers[handle] = null;
+  }
+
+  public destroy(): void {
+    if (this.adopted) return;
+    BackendContext.module.GL.deleteContext(this.handle);
+  }
+
+  private register<T>(table: (T | null)[], item: T): number {
+    const handled = BackendContext.module.GL.getNewId(table);
```
Comment (domchen) [inline web/src/core/backend-context.ts:107]: handled->handle

File: web/src/core/backend-context.ts (hunk 1 above)
Comment (zenoslin) [reply web/src/core/backend-context.ts:79]: 这里是参考了EmscriptenGL里的deleteContext，用unregisterTexture确实语义化会好一点

**Review by lvpengwei — APPROVED** [review]

**Pull request merged**